lte/gateway/c/oai/tasks/gtpv1-u/gtp_tunnel_openflow.c

```diff
@@ -382,10 +383,31 @@ static const struct gtp_tunnel_ops openflow_ops = {
     .get_dev_name           = openflow_get_dev_name,
 };
 
+static const struct gtp_tunnel_ops upf_openflow_ops = {
+    .init                   = openflow_init,
+    .uninit                 = openflow_uninit,
+    .reset                  = openflow_reset,
+    .add_tunnel             = upf_add_tunnel,
+    .del_tunnel             = upf_del_tunnel,
+    .discard_data_on_tunnel = upf_discard_data_on_tunnel,
+    .forward_data_on_tunnel = upf_forward_data_on_tunnel,
+    .add_paging_rule        = upf_add_paging_rule,
+    .delete_paging_rule     = upf_delete_paging_rule,
+    .send_end_marker        = openflow_send_end_marker,
```

Comment:
can you define method in UPF for this using RYU APIs.

Reply:
Comments addressed.  The place holder is added. The mapping of call to the end ryu APIs will be done in #5685 which has the changes for flow_dl as well.

Reply:
Yes. It was initially planned for #5685. Idea was to rebase it once #5217 gets merged. But as #5685 is also almost ready it will be better to track it with new PR.